src/scripts/components/ProfileCuration.js

```diff
@@ -0,0 +1,522 @@
+  flex-direction: column;
+  justify-items: center;
+  padding: 24px 14px 14px;
+  text-align: center;
+`
+
+const VotingBarWrapper = styled(ProgressBarWrapper)`
+  background: ${props => props.theme.colors.ProfileCuration.VotingBarTwo};
+  margin: 20px 0 25px;
+`
+
+const VotingBar = styled(ProgressBar)`
+  background: ${props => props.theme.colors.ProfileCuration.VotingBarOne};
+`
+
+const ChallengePeriod = styled.div`
+  align-items: center;
+  background: linear-gradient(
+    135deg,
+    ${props =>
+    props.inReveal
+      ? props.theme.colors.ProfileCuration.ChallengeBackgroundTwoFrom
+      : props.theme.colors.ProfileCuration.ChallengeBackgroundOneFrom}
+      0%,
+    ${props =>
+    props.inReveal
+      ? props.theme.colors.ProfileCuration.ChallengeBackgroundTwoTo
+      : props.theme.colors.ProfileCuration.ChallengeBackgroundOneTo}
+      100%
+  );
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: ${INFOSTATUS_CARD_MARGIN_BOTTOM};
+  padding: 50px 25px 60px;
+`
+
+const ChallengeTimeWrapper = styled.div`
+  margin: 35px 0;
+  text-align: center;
+  width: 100%;
+`
+
+const ChallengeTime = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+  text-align: center;
+  vertical-align: bottom;
+  width: 16%;
+`
+
+const ChallengeSequence = styled(ChallengeTimeWrapper)`
+  margin: 0;
+`
+
+const ChallengeSequenceText = styled(Text)`
+  display: inline-block;
+  text-align: center;
+  width: 33.2%;
+`
+
+const ChallengeTimeline = styled(ChallengeTimeWrapper)`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 0 0;
+  position: relative;
+  width: 70%;
+`
+
+const ChallengeLine = styled.span`
+  border-top: 2px ${({ dashed }) => (dashed ? 'dashed' : 'solid')}
+    ${props => props.theme.colors.ProfileCuration.ChallengeSequenceDot};
+  left: 0;
+  opacity: ${({ dashed }) => (dashed ? 0.5 : null)};
+  opacity: ${({ dashed }) => (dashed ? 0.5 : null)};
+  position: absolute;
+  top: 50%;
+  transform: translate3d(0, -50%, 0);
+  width: ${({ width }) => width || '100%'};
+`
+
+const DotActive = css`
+  &::before,
+  &::after {
+    content: '';
+  }
+`
+
+const ChallengeSequenceDot = styled.span`
+  background: ${props =>
+    props.theme.colors.ProfileCuration.ChallengeSequenceDot};
+  border-radius: 100%;
+  height: ${props => (props.active ? '12px' : '8px')};
+  position: relative;
+  width: ${props => (props.active ? '12px' : '8px')};
+  z-index: 2;
+  ${props => (props.active ? DotActive : null)} &::before, &::after {
+    border: 2px solid
+      ${props => props.theme.colors.ProfileCuration.ChallengeSequenceDot};
+    border-radius: 100%;
+    left: 50%;
+    position: absolute;
+    top: 50%;
+    transform: translate3d(-50%, -50%, 0);
+  }
+  &::before {
+    height: 22px;
+    opacity: 0.5;
+    width: 22px;
+  }
+  &::after {
+    height: 36px;
+    opacity: 0.2;
+    width: 36px;
+  }
+`
+
+class Profile extends Component<Props, void> {
```

Comment:
If the component takes no props you can probably delete `Props` and just do `Component<void>`

Reply:
Done!